### PR TITLE
added cerberus recipe

### PR DESCRIPTION
**Pull request by kyleniemeyer**
Added a new recipe for [Cerberus](http://docs.python-cerberus.org/).

### CODE REVIEW

File: recipes/cerberus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  imports:
+    - cerberus
+    - cerberus.tests
+
+  commands:
+      - python -m unittest -v cerberus.tests.tests
+
+about:
+  home: https://github.com/nicolaiarocci/cerberus
+  license: ISC License (ISCL)
```
Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: ISC is sufficient.

Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: Also please add `license_file: LICENSE`.

File: recipes/cerberus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
```
Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: This comment can be dropped.

File: recipes/cerberus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.9.2" %}
+{% set sha256 = "b122c7b2cbf856ea2587e187fac968fc8dcd49d47aa1f239abd9eaa0ed86a7ce" %}
+
+package:
+  name: cerberus
+  version: {{ version }}
+
+source:
+  fn: Cerberus-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/c1/7a/65f3aa48279cda81208ccca4c932e63fedaf02f80f1fb6a482a7b8d8f239/Cerberus-0.9.2.tar.gz
```
Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: Please use the [templated url]( https://github.com/conda-forge/staged-recipes/blob/3080b8948ce291964824b04ad01e5b68186af404/recipes/example/meta.yaml#L18 ) from the example.

File: recipes/cerberus/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.9.2" %}
```
Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: Please add `{% set name = "Cerberus" %}` at the top.

File: recipes/cerberus/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+{% set sha256 = "b122c7b2cbf856ea2587e187fac968fc8dcd49d47aa1f239abd9eaa0ed86a7ce" %}
+
+package:
+  name: cerberus
```
Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: Please change to `{{ name|lower }}`.

File: recipes/cerberus/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  fn: Cerberus-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/cerberus/meta.yaml]: Please change to `{{ name }}-{{ version }}.tar.gz`.

**Comment by jakirkham** [comment]
Thanks @kyleniemeyer. Few simple comments to fix above. Taking a look at the example recipe included should help.

**Comment by kyleniemeyer** [comment]
Thanks for the feedback @jakirkham, I've made all of those changes.

**Comment by jakirkham** [comment]
Thanks. LGTM.

**Comment by jakirkham** [comment]
Thanks @kyleniemeyer.

**Comment by jakirkham** [comment]
Soon you should get an email that invites you to join conda-forge, @kyleniemeyer. Once accepted you will be added to a team with the same name as this recipe. Those will give you permissions on the feedstock (repo) for this recipe.

Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make. Please let us know if you have any questions and welcome to conda-forge.

**Comment by kyleniemeyer** [comment]
Great, thanks!

**Pull request merged**